### ISSUE TITLE
Calculation of sharp cut off of organic aerosols in ELMO

### ISSUE THREAD

**Issue by bnmurphy** [USEPA/CMAQ#212]
**Description**
The calculation of organic aerosol from the ELMO module in v5.4 has slight errors for sharp cut offs. The calculation of total PMF organic aerosol is correct.

**Scope and Impact**
This effects calculations of organic aerosol for PM2.5 and PM1 with the hard size cutoff.

**Solution**
A permanent solution will be posted via Pull Request to this repository. Temporarily, users can fix the issue in ELMO_PROC.F:
Change
```
Line 2707: CALL GET_AERO_INLET( IPM1, IM, IWET, FRAC( IMODE ) )
Line 2723: CALL GET_AERO_INLET( IPM1, IM, IWET, FRAC( IMODE ) )
Line 2736: CALL GET_AERO_INLET( IPM25, IM, IWET, FRAC( IMODE ) )
Line 2752: CALL GET_AERO_INLET( IPM25, IM, IWET, FRAC( IMODE ) )
```
To
```
Line 2707: CALL GET_AERO_INLET( IPM1, IMODE, IWET, FRAC( IMODE ) )
Line 2723: CALL GET_AERO_INLET( IPM1, IMODE, IWET, FRAC( IMODE ) )
Line 2736: CALL GET_AERO_INLET( IPM25, IMODE, IWET, FRAC( IMODE ) )
Line 2752: CALL GET_AERO_INLET( IPM25, IMODE, IWET, FRAC( IMODE ) )
```

**Comment by kmfoley** [comment]
This bugfix was included in on the v5.4+ branch [(CMAQv5.4.0.4)](https://github.com/USEPA/CMAQ/releases/tag/CMAQv5.4.0.4_9April2024) on April 9, 2024, and will be released with CMAQv5.5.

**Comment by bnmurphy** [comment]
Addressed in PR #218
